### ISSUE TITLE
Unable to build on s390x

### ISSUE THREAD

**Issue by Masquerade0097** [ianlancetaylor/cgosymbolizer#11]
I get the following error when I try to build the package on IBM Z (s390x) machine.

```
# github.com/rai-project/tensorflow/vendor/github.com/ianlancetaylor/cgosymbolizer
In file included from dwarf.c:33:0:
config.h:9:2: error: #error BACKTRACE_ELF_SIZE unknown
 #error BACKTRACE_ELF_SIZE unknown
  ^~~~~
```

 Can anyone please let me know the fix for it?

**Comment by ianlancetaylor** [comment]
I thought this was fixed by d43e30eacb43da92f28d523438f977af0a7b6072.  Do you have that in your copy?

**Comment by Masquerade0097** [comment]
I updated my copy to point to the latest commit and that fixed the issue. Thanks.
